requirements.txt

```diff
@@ -55,7 +55,6 @@ markupsafe==1.0           # via jinja2
 monotonic==1.3            # via oslo.utils
 more-itertools==4.3.0     # via pytest
 msgpack-python==0.4.8     # via oslo.serialization
-mysqlclient==1.4.2
```

Comment:
This is not related change.

Reply:
It's done automatically by `pip-compile`. It's happening because there is no mysql listed in `install_requires` but it's used in `teuthology/coverage.py`.